### DISCUSSION TITLE
make components more interactive

### DISCUSSION THREAD

**Discussion by BanDroid** [saadeghi/daisyui#2857] [Ideas / feature requests]
it seems the components are not very interactive compared to other tailwind plugin. for example, the button now is really lack of animation or transition, use the black theme, the button only show as a square block, no animation or transition on hover, active, or anything, it looks bad in default config.

**Comment by NikarashiHatsu** [comment]
Have inspirations? Make a PR and we'll review it as soon as possible instead of cursing it 😄

**Reply by BanDroid** [reply]
sorry, i'm not cursing or anything, just starting a discussion in case anyone interested. once i read the codebase, i might start to contribute as well.

**Comment by saadeghi** [comment]
> use the black theme, the button only show as a square block, no animation

There are all the other themes you can use. Some people prefer this specific theme, some don't.  
Plus, you can customize the themes the way you want (you can even change the animations)  
Plus, you can make a new custom theme easily.

**Reply by BanDroid** [reply]
the black theme just an example, i just pointing out there are some components need more interactivity, i'll try to contribute once i know the full code.

i also know how to customize it, but it is not practical to repeat the same process in each application if it's the same cutomization.

**Reply by BanDroid** [reply]
i've checked the old v2 version, the components seems having better interactivity than the new one
